crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -149,6 +152,20 @@ def cancel_allowed?
       ].include?(current_step) && !running?(:admin)
     end
 
+    def save_crowbar_backup(backup_location = "")
```

Comment:
+1 on not giving the default. This is a clear indication to the caller that something needs to be passed.